packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-self-reg/main/java/integrationsteps.md

```diff
@@ -1,5 +1,3 @@
-## Integration steps
-
 ### Step 1: Register new users
```

Comment:
The Steps in java self-reg use-case still use "Step X:" format. I think they should be removed. (If Van is taking care of this separately, ignore.)

Reply:
Yeah - for self-reg - I was waiting until everyone was done reordering before I went in and updated the headings. It's also ok if each of us update those ourselves...but might be better for consistency if just one of us (me) does it. Thank you!